src/App.tsx

```diff
@@ -26,69 +26,112 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+  const visibleGoods: string[] = [...goods];
+
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case 1:
```

Comment:
тут краще використовувати enum
```suggestion
      case SortType.ALPHABET:
```